snap/backend/src/utils/transactionUtils.js

```diff
@@ -0,0 +1,157 @@
+import symbolClient from '../services/symbolClient.js';
+import moment from 'moment'; // eslint-disable-line
+import { PublicKey } from 'symbol-sdk';
+import { SymbolFacade } from 'symbol-sdk/symbol';
+
+const AGGREGATE_COMPLETE_TRANSACTION_TYPE = 16705;
+const AGGREGATE_BONDED_TRANSACTION_TYPE = 16961;
+const TRANSFER_TRANSACTION_TYPE = 16724;
+
+const TRANSACTION_TYPE = {
```

Comment:
Should be able to get the transactions type from the SDK - ``models.TransactionType``